src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,30 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws MyRegistrationException {
```

Comment:
I can't see the first check. Solve task according to the requirenments and request new review.
![image](https://user-images.githubusercontent.com/20326858/216309234-34b905d2-064b-4f11-bbf7-93a4dc10049e.png)